librina installation fails because AC_PATH_PROG  of PROTOC is not being checked

Hi,

I am trying to install the sdk and I fell into this error:
```
rm -f CDAP.tmp
touch CDAP.tmp
/bin/false -I. -I/home/bird/irati/librina/src	\
	--cpp_out=.		\
	/home/bird/irati/librina/src/CDAP.proto
Makefile:1695: recipe for target 'CDAP.stamp' failed
```
This is caused because during configuration the check of the PROTOC path is failing:
```
checking for management_sdu_write syscall availability... yes
checking for protoc... /bin/false
checking for LIBPROTOBUF... yes
```
The root problem seems to be that the assigned path is not being checked:
https://github.com/IRATI/stack/blob/master/librina/configure.ac#L240

Other paths that are not being checked are:
https://github.com/IRATI/stack/blob/master/rinad/configure.ac#L28-L31
https://github.com/IRATI/stack/blob/master/rina-tools/configure.ac#L31-L34
https://github.com/IRATI/stack/blob/master/librina/configure.ac#L32-L35

--- a/librina/configure.ac
+++ b/librina/configure.ac
@@ -29,10 +29,22 @@ AC_PROG_GREP
 AC_PROG_MKDIR_P
 AC_PROG_LN_S
 
-AC_PATH_PROG([FMT],  [fmt],  [/bin/false])
-AC_PATH_PROG([SORT], [sort], [/bin/false])
-AC_PATH_PROG([UNIQ], [uniq], [/bin/false])
-AC_PATH_PROG([GIT],  [git],  [/bin/false])
+AC_PATH_PROG([FMT],  [fmt],  [no])
+AS_IF([test "x$FMT" = "xno"],[
+	AC_MSG_ERROR([Your system lacks of fmt support])
+])
+AC_PATH_PROG([SORT], [sort], [no])
+AS_IF([test "x$SORT" = "xno"],[
+	AC_MSG_ERROR([Your system lacks of sort support])
+])
+AC_PATH_PROG([UNIQ], [uniq], [no])
+AS_IF([test "x$UNIQ" = "xno"],[
+	AC_MSG_ERROR([Your system lacks of uniq support])
+])
+AC_PATH_PROG([GIT],  [git],  [no])
+AS_IF([test "x$GIT" = "xno"],[
+	AC_MSG_ERROR([Your system lacks of git support])
+])
 
 AC_PROG_CPP
 AC_PROG_CC
@@ -241,7 +253,10 @@ AX_HAVE_SYSCALL([management_sdu_write],[],[
 CFLAGS="$oldcflags"
 AC_LANG_POP([C])
 
-AC_PATH_PROG([PROTOC], [protoc], [/bin/false])
+AC_PATH_PROG([PROTOC], [protoc], [no])
+AS_IF([test "x$PROTOC" = "xno"],[
+	AC_MSG_ERROR([Your system lacks of protoc support])
+])
 
 LIBPROTOBUF_MIN_VERSION=2.5.0
 AC_SUBST(LIBPROTOBUF_MIN_VERSION, $LIBPROTOBUF_MIN_VERSION)

--- a/rina-tools/configure.ac
+++ b/rina-tools/configure.ac
@@ -28,10 +28,22 @@ AC_PROG_GREP
 AC_PROG_MKDIR_P
 AC_PROG_LN_S
 
-AC_PATH_PROG([FMT],  [fmt],  [/bin/false])
-AC_PATH_PROG([SORT], [sort], [/bin/false])
-AC_PATH_PROG([UNIQ], [uniq], [/bin/false])
-AC_PATH_PROG([GIT],  [git],  [/bin/false])
+AC_PATH_PROG([FMT],  [fmt],  [no])
+AS_IF([test "x$FMT" = "xno"],[
+	AC_MSG_ERROR([Your system lacks of fmt support])
+])
+AC_PATH_PROG([SORT], [sort], [no])
+AS_IF([test "x$SORT" = "xno"],[
+	AC_MSG_ERROR([Your system lacks of sort support])
+])
+AC_PATH_PROG([UNIQ], [uniq], [no])
+AS_IF([test "x$UNIQ" = "xno"],[
+	AC_MSG_ERROR([Your system lacks of uniq support])
+])
+AC_PATH_PROG([GIT],  [git],  [no])
+AS_IF([test "x$GIT" = "xno"],[
+	AC_MSG_ERROR([Your system lacks of git support])
+])
 
 AC_PROG_CPP
 AC_PROG_CC

--- a/rinad/configure.ac
+++ b/rinad/configure.ac
@@ -25,10 +25,22 @@ AC_PROG_GREP
 AC_PROG_MKDIR_P
 AC_PROG_LN_S
 
-AC_PATH_PROG([FMT],  [fmt],  [/bin/false])
-AC_PATH_PROG([SORT], [sort], [/bin/false])
-AC_PATH_PROG([UNIQ], [uniq], [/bin/false])
-AC_PATH_PROG([GIT],  [git],  [/bin/false])
+AC_PATH_PROG([FMT],  [fmt],  [no])
+AS_IF([test "x$FMT" = "xno"],[
+	AC_MSG_ERROR([Your system lacks of fmt support])
+])
+AC_PATH_PROG([SORT], [sort], [no])
+AS_IF([test "x$SORT" = "xno"],[
+	AC_MSG_ERROR([Your system lacks of sort support])
+])
+AC_PATH_PROG([UNIQ], [uniq], [no])
+AS_IF([test "x$UNIQ" = "xno"],[
+	AC_MSG_ERROR([Your system lacks of uniq support])
+])
+AC_PATH_PROG([GIT],  [git],  [no])
+AS_IF([test "x$GIT" = "xno"],[
+	AC_MSG_ERROR([Your system lacks of git support])
+])
 
 AC_PROG_CXXCPP
 AC_PROG_CXX

